prob03-simple-db/simpleDB/simpleDBTest/DataCommandTest.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using NUnit.Framework;
+using simpleDB;
+using System.Collections.Generic;
+
+namespace simpleDBTest
+{
+    [TestFixture]
+    public class DataCommandTest
+    {
+        //should never need to reinstantiate the data commands
+        Set set = new Set();
+        Get get = new Get();
+        NumEqualTo numEqualTo = new NumEqualTo();
+        Unset unSet = new Unset();
+        ResetAll reset = new ResetAll();
+
+        [SetUp]
+        public void SetUp()
+        {
+            reset.Operation(null);
+        }
+
+        [Test]
+        public void Canary()
+        {
+            Assert.IsTrue(true);
+        }
+
+        [Test]
+        public void TestSetExisting()
+        {
+            List<string> arg1 = new List<string>(){
+                "z",
+                "23"
+            };
+            List<string> arg2 = new List<string>(){
+                "z",
+                "9"
+            };
+
+            set.Operation(arg1);
+            set.Operation(arg2);
+            int? actual = get.Operation(arg1);
+
+            Assert.AreEqual(9, actual);
+        }
+
+        [Test]
+        public void TestSetAndGet()
+        {
+            List<string> args = new List<string>(){
+                "a",
+                "10"
+            };
+
+            set.Operation(args);
+            int? actual = get.Operation(args);
```

Comment:
well I don't see it as a problem if you pass more arguments than needed, they just get ignored. I can't think of a scenario where that would be problematic. Correct me if I'm wrong